examples/pytorch/hilander/PSS/Smooth_AP/src/evaluate.py

```diff
@@ -0,0 +1,377 @@
+# repo originally forked from https://github.com/Confusezius/Deep-Metric-Learning-Baselines
+
```

Comment:
I see that neither main.py and finetune_1head.py uses auxiliaries_nofaiss.py, rather, both use auxiliaries.py, can we remove this file?

Reply:
Yes. auxiliaries_nofaiss.py removed.